tests/unit/amo/reducers/test_home.js

```diff
@@ -372,10 +372,24 @@ describe(__filename, () => {
           },
         },
         secondary: {
-          cta: heroShelves.secondary.cta,
-          description: heroShelves.secondary.description,
-          headline: heroShelves.secondary.headline,
-          modules: heroShelves.secondary.modules,
+          cta: {
+            url: heroShelves.secondary.cta.url,
+            outgoing: heroShelves.secondary.cta.outgoing,
+            text: heroShelves.secondary.cta.text[lang],
+          },
```

Comment:
We usually use the `createInternalXXX()` function instead of defining the actual object:

```suggestion
          cta: createInternalHeroCallToAction(heroShelves.secondary.cta, lang),
```

Reply:
https://github.com/mozilla/addons-frontend/pull/10086/commits/7d92b37c782685e84da506439bf1f1cf070b5806